### PR TITLE
Epic/style guide edits for jupiter

### PR DESCRIPTION
**Pull request by alexisBimatovs**
screen shot 2017-09-06 at 9 40 40 am

### CODE REVIEW

**Comment by schweitzdesign** [comment]
Marked this on hold as we don't want to merge to dev until we're ready to deploy jupiter

File: app/helpers/forever_style_guide/application_helper.rb [hunk 1]
```diff
@@ -265,40 +261,24 @@ def new_digital_art_url
...
-    def wall_art_url
-      www_url('/wall_art')
+    def free_digital_art_url
+      www_url('/digital_art_library?facet_name=free')
     end
 
-    def photo_gifts_url
-      www_url('/photo_gifts')
+    def predesigned_layouts_digital_art_url
+      www_url('/digital_art_library?facet_name=page-layouts')
```
Comment (schweitzdesign) [inline app/helpers/forever_style_guide/application_helper.rb]: I think this should be more like this
www.forever.com/digital_art_library?facet_name=pre-designed%20layouts

File: app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb [hunk 2]
```diff
@@ -42,81 +42,70 @@
...
           <li>
-            <%= link_to trademark('pixels2Pages'), p2p_url, class: 'dropdown-menu-nav-link', title: 'Premium Training and Resources' %>
-            <small class="dropdown-menu-nav-link-subhead">Premium Training and Resources</small>
+            <%= link_to trademark('Forever Artisan'), artisan_url, class: 'dropdown-menu-nav-link', title: 'Forever Artisan 5 Digital Scrapbooking Software' %>
+              <small class="dropdown-menu-nav-link-subhead">Windows Scrapbooking Software</small>
           </li>
 
+          <ul class="list-unstyled dropdown-menu-nav-bullets">
+            <li>
+              <%= link_to '- Photo Books, Cards, Calendars, & More', new_digital_art_url, class: 'dropdown-menu-nav-link', title: 'Artisan Photo Books, Cards, Calendars, & More' %>
```
Comment (schweitzdesign) [inline app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb]: new_digital_art_url should not be here. I think the intent was to link to an anchor tag on the /artisan page. Can you confirm an id will be available to link to that way @amiballo

Comment (alexisBimatovs) [reply app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb]: yes was working on this now, I had not assigned yet cause I was ironing some of these details out, sorry for confusion prob could have waited to push up :)

File: app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss [hunk 3]
```diff
@@ -206,3 +206,7 @@ a.dropdown-menu-nav-link {
     font-size: $font-size-500;
   }
 }
+.navbar-new_tag {
+  vertical-align: 10px; // added for superscript styling
+  margin-left: 0;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss]: this margin property is unnecessary

File: app/helpers/forever_style_guide/application_helper.rb (hunk 1 above)
Comment (alexisBimatovs) [reply app/helpers/forever_style_guide/application_helper.rb]: I was following the convention of using the name as in other references here but checked in with Brad and it is preferred to use the slug so this has been updated

Comment (schweitzdesign) [reply app/helpers/forever_style_guide/application_helper.rb]: the name would work, but there is no facet named page layouts
https://www.forever.com/digital_art_library?facet_name=page-layouts

**Comment by schweitzdesign** [comment]
Looks good thanks. We'll keep this on hold and can branch off accordingly / as necessary if there are further changes.

Please also open a PR against the store that uses the styleguide off of this branch.

**Comment by schweitzdesign** [comment]
for reference: https://github.com/forever-inc/a-new-hope/commit/e14e31802a7077bab9106a4254fc333eb5f6eb6b

**Comment by alexisBimatovs** [comment]
https://github.com/forever-inc/a-new-hope/pull/789

File: app/helpers/forever_style_guide/application_helper.rb [hunk 4]
```diff
@@ -265,40 +261,28 @@ def new_digital_art_url
...
-    def wall_art_url
-      www_url('/wall_art')
+    def free_digital_art_url
+      www_url('/digital_art_library?facet_name=free')
     end
 
-    def photo_gifts_url
-      www_url('/photo_gifts')
+    def predesigned_layouts_digital_art_url
+      www_url('/prints_library?facet_name=pre-designed%20layouts')
```
Comment (schweitzdesign) [inline app/helpers/forever_style_guide/application_helper.rb]: should be digital_art_library

**Comment by alexisBimatovs** [comment]
https://github.com/forever-inc/a-new-hope/pull/790

@schweitzdesign

**Pull request merged**